packages/modals/src/elements/TooltipModal/TooltipModal.tsx

```diff
@@ -0,0 +1,215 @@
+   * The root ID to use for descendants. A unique ID is created if none is provided.
+   **/
+  id?: string;
+}
+
+export const TooltipModal = React.forwardRef<HTMLDivElement, ITooltipModalProps>(
+  (
+    {
+      referenceElement,
+      popperModifiers,
+      placement,
+      onClose,
+      hasArrow,
+      isAnimated,
+      zIndex,
+      style,
+      backdropProps,
+      focusOnMount,
+      restoreFocus,
+      id,
+      ...props
+    },
+    ref
+  ) => {
+    const theme = useContext(ThemeContext);
+    const previousReferenceElementRef = useRef<HTMLElement | null>();
+    const modalRef = useCombinedRefs(ref);
+    const [popperElement, setPopperElement] = useState<HTMLDivElement | null>();
+    const {
+      getTitleProps,
+      getCloseProps,
+      getContentProps,
+      getBackdropProps,
+      getModalProps
+    } = useModal({
+      id,
+      onClose,
+      modalRef,
+      focusOnMount,
+      /** Handle `restoreFocus` locally to return focus to `referenceElement` */
+      restoreFocus: false
+    });
+
+    useEffect(() => {
+      if (!referenceElement && previousReferenceElementRef.current && restoreFocus) {
+        previousReferenceElementRef.current.focus();
+      }
+
+      previousReferenceElementRef.current = referenceElement;
+    }, [referenceElement, restoreFocus]);
+
+    const popperPlacement = useMemo(
+      () => (theme.rtl ? getRtlPopperPlacement(placement!) : getPopperPlacement(placement!)),
+      [placement, theme.rtl]
+    );
+
+    const { styles, attributes, state } = usePopper(referenceElement, popperElement, {
+      placement: popperPlacement,
+      modifiers: [
+        { name: 'offset', options: { offset: [0, theme.space.base * 2] } }, // Default Popper offset
+        ...(popperModifiers || [])
+      ]
+    });
+
+    const value = {
+      getTitleProps,
+      getContentProps,
+      getCloseProps
+    };
+
+    const modalProps = getModalProps({
+      ref: modalRef,
+      placement: state ? state.placement : 'top',
+      hasArrow,
+      isAnimated,
+      style,
+      ...props
+    }) as any;
+
+    return referenceElement ? (
+      <TooltipModalContext.Provider value={value}>
+        <StyledTooltipModalBackdrop {...(getBackdropProps(backdropProps) as any)}>
+          <StyledTooltipWrapper
+            ref={setPopperElement}
+            style={styles.popper}
+            placement={state ? state.placement : undefined}
+            zIndex={zIndex}
+            isAnimated={isAnimated}
+            {...attributes.popper}
+          >
+            <StyledTooltipModal {...modalProps} />
+          </StyledTooltipWrapper>
+        </StyledTooltipModalBackdrop>
+      </TooltipModalContext.Provider>
+    ) : null;
+  }
+) as IStaticTooltipModalExport<HTMLDivElement, ITooltipModalProps>;
+
+TooltipModal.Title = Title;
+TooltipModal.Body = Body;
+TooltipModal.Close = Close;
+TooltipModal.Footer = StyledTooltipModalFooter;
+TooltipModal.FooterItem = StyledTooltipModalFooterItem;
```

Comment:
Looks like some of these are still showing up in Styleguidist. Should they be removed?

Reply:
I've updated the glob to remove these.